ethereum/contracts/zksync/libraries/Diamond.sol

```diff
@@ -129,7 +129,7 @@ library Diamond {
     ) private {
         DiamondStorage storage ds = getDiamondStorage();
 
-        require(_facet != address(0), "G"); // facet with zero address cannot be added
+        require(_facet.code.length > 0, "G"); // facet with no code cannot be added
```

Comment:
Please write a comment that this check also verify that `_facet` is not a zero address. Zero address is a special here, since it is an address that empty selector associated with.